update-overlay/action.yaml

```
@@ -17,6 +17,10 @@ inputs:
     description: ""
     required: true
 
+  match-type:
+    description: ""
```

please provide a description
```suggestion
    description: "can be either 'exact' or 'best-effort'"
```